.github/workflows/test.yml

```diff
@@ -37,7 +37,6 @@ jobs:
         include:
           - container: ghcr.io/oca/oca-ci/py3.10-odoo17.0:latest
             include: "database_cleanup"
-            makepot: "true"
```

Comment:
Removing this one, no POT will be generated for database_cleanup AFAIK.

Reply:
Ah sorry, it does say 'for installed addons' in that first link, so you are right. Will amend.